### PR TITLE
permission denied due to missing docker login

### PR DESCRIPTION
**Pull request by nidve01**

### CODE REVIEW

File: register-qubeship.sh [hunk 1]
```diff
@@ -35,6 +35,15 @@ else
     exit -1
 fi
 
+if [ $is_beta ];  then
+    echo "docker login"
```
Comment (erioness1125) [inline register-qubeship.sh]: let's delete this line or replace it with more informative message like: "login to Qubeship docker registry..." (or any better message)

Comment (nidve01) [inline register-qubeship.sh]: Done. Thanks!

File: register-qubeship.sh [hunk 2]
```diff
@@ -35,6 +35,15 @@ else
...
+if [ $is_beta ];  then
+    echo "login to Qubeship docker registry"
+    docker login -u $BETA_ACCESS_USERNAME -p $BETA_ACCESS_TOKEN quay.io
+    if [ $? -ne 0 ]; then
+        echo "ERROR : failed to do docker login. please check your docker installation"
+        exit 1
+    fi
+fi
+
 if [ $auto_pull ] ; then
```
Comment (erioness1125) [inline register-qubeship.sh]: I don't think we need this if clause here... as we have another command `docker-compose $files pull oauth_registrator` below, it should be okay to delete it.

**Comment by erioness1125** [comment]
tested and worked well 👍

**Comment by nidve01** [comment]
Thanks

**Pull request merged**